src/esn.calendar.libs/app/services/partstat-jwt.service.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const jwtDecode = require('jwt-decode');
+
+angular.module('esn.calendar.libs')
+  .service('calPartstatJwtService', calPartstatJwtService);
+
+function calPartstatJwtService($q, $http, $log, httpConfigurer, calEventService, calendarHomeService, calOpenEventForm, notificationFactory, calPartstatUpdateNotificationService) {
```

Comment:
We should uppercase for all charaters of JWT

Reply:
For me, `PartstatJwtService`  is a not good name since JWT is a mechanism for this feature.

Incase you want to keep it, the jsdoc to explain service is necessary.